pgxn/neon/communicator/build.rs

```diff
@@ -0,0 +1,22 @@
+use std::env;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let crate_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
+
+    cbindgen::generate(crate_dir).map_or_else(
```

Comment:
nit: consider just using a `match` here instead of `map_or_else()`, including the `Err(ParseSyntaxError)` pattern -- we're not using the final result for anything so there's no point in mapping it, and a `match` reads better.